lib/src/layer/tile_layer/tile.dart

```diff
@@ -21,13 +21,21 @@ class Tile extends StatefulWidget {
   /// visible pixel when the map is rotated.
   final Point<double> currentPixelOrigin;
 
+  /// Position Coordinates.
+  ///
+  /// Most of the time, they are the same as in [tileImage].
+  /// Except for multi-world maps.
+  /// TODO replace tileImage + positionCoordinates with tileRenderer?
```

Comment:
I think this could do with a little better explanation, maybe an example?

Reply:
I've just added as comments the example about Alaska.